components/org.wso2.carbon.crypto.impl/src/main/java/org/wso2/carbon/crypto/impl/DefaultCryptoService.java

```diff
@@ -541,8 +541,69 @@ public PrivateKey getPrivateKey(CryptoContext cryptoContext) throws CryptoExcept
 
     }
 
-    // ------------ Management methods of the default crypto service starts here. --------------------
+    @Override
+    public byte[] encrypt(byte[] cleartext, String algorithm, String javaSecurityAPIProvider,
+                          boolean returnSelfContainedCipherText, String internalCryptoProviderType)
+            throws CryptoException {
+
+        failIfInternalCryptoInputsAreNotValid(cleartext, algorithm, internalCryptoProviderType,
+                "'Internal Encryption'");
+        if (log.isDebugEnabled()) {
+
```

Comment:
Remove unnecessary new lines in the debug block. Check the other places in this class as well.

Reply:
Fixed with 8d41f6ac26b91bf9356a2f68ca4d974e3997a10a